src/pages/[platform]/get-started/tutorials/connect-api-and-database.mdx

```diff
@@ -2,7 +2,8 @@ import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
 
 export const meta = {
   title: 'Connect API and Database',
-  description: 'Connect API and Database',
+  description:
+    'Connect API and Database The Amplify open-source client libraries provide use-case centric, and easy-to-use interfaces across different categories of AWS cloud powered operations enabling mobile and web developers to easily interact with their backends. The Amplify libraries can be used with both new backends created using the Amplify CLI or existing backend resources.',
```

Comment:
Setting this up as an example of a really long description to make use of `-webkit-line-clamp`: https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-line-clamp